site/content/docs/getting-further/automation/automation-options.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Automation Guide - Options"
+tags: 
+- automation
+- guide
+type: page
+---
+
+If you want to use ZAP for automated security scanning then you have a wide range of options, also listed on the main 
+[Automate](http://localhost:3000/docs/automate/) page.
+
+Each of these options provides a different balance between ease of use and flexibility + functionality:
+
+| Option                                        | Ease of Use     | Flexibility / Functionality     |
+|-----------------------------------------------|-----------------|-----------------|
+| [ZAPit](#zapit)                               | 🟢 🟢 🟢 🟢 🟢 | 🔴              |
+| [Quick Start](#quick-start-command-line)      | 🟢 🟢 🟢 🟢 🟢 | 🔴              |
+| [Packaged Scans](#docker-packaged-scans)      | 🟢 🟢 🟢 🟢    | 🟡 🟡 🟡        |
+| [GitHub Actions](#github-actions)             | 🟢 🟢 🟢 🟢    | 🟡 🟡 🟡        |
+| [Automation Framework](#automation-framework) | 🟢 🟢 🟢 🟡    | 🟢 🟢 🟢 🟢     |
+| [API](#api-and-daemon-mode)                   | 🟠 🟠          | 🟢 🟢 🟢 🟢 🟢 |
+
+&nbsp;
+
+These automation options also have different focuses:
+
+### ZAPit
+
+[ZAPit](/blog/2023-10-18-zapit/) provides a quick ‘reconnaissance’ scan of the URL specified via the command line. 
+It is safe to run against production services but is very limited in its scope as it currently only checks the 
+URL you specify and does not explore the app any further.
+
+### Quick Start command line
+
+The [Quick Start command line](/docs/desktop/addons/quick-start/cmdline/) option is easy to run, but with very limited options so only suitable for simple scans.
+ZAP will run the standard [Spider]() to explore the site and then run the Active Scanner with the default scan policy.
+
+### Docker Packaged Scans
+The [Docker Packaged Scans](/docs/docker/) are the easiest way to get started with ZAP automation and provide a reasonable amount of flexibility.
+They allow you to configure the most commonly used option via the command line and allow you to make ZAP API calls at each stage of the scan via [Scan Hooks](/docs/docker/scan-hooks/).
+
+If you need to get ZAP to handle authentication for your app then the Automation Framework is likely to be a better option for you.
+The Packaged Scans only run in Docker containers and are being migrated to use the Automation Framework under the covers.
+
+### GitHub Actions
+The [GitHub Actions](https://github.com/marketplace?query=zap&verification=verified_creator) are the Packaged Scans 
+made available on the GitHub Marketplace.
+They also provide the option to raise issues directly in GitHub for any potential vulnerabilities found.
```

Comment:
> on GitHub

Reply:
Missed

Reply:
Ah! I had changed "on the GitHub Marketplace" to "on GitHub.." 😉

Reply:
Oops, yup!